### PR TITLE
Revise greater equal

### PR DESCRIPTION
**Pull request by pszmel**
### Details:
- update docs
- update backend test
- add op to constants.py

### Tickets:
 - 37442

### CODE REVIEW

**Review by dkozykowski — COMMENTED** [review]
You may add broadcast type information in *GreaterEqual_1.md* in lines 45 and 68 as it was done in this PR: #6913


Also I think those two fragments could be removed, as this function is inherited from base class:
https://github.com/openvinotoolkit/openvino/blob/600eef24bca97fb049fc580c2d02f6982050ff6c/ngraph/core/src/op/greater_eq.cpp#L90-L94
https://github.com/openvinotoolkit/openvino/blob/600eef24bca97fb049fc580c2d02f6982050ff6c/ngraph/core/include/ngraph/op/greater_eq.hpp#L27

File: docs/template_plugin/tests/functional/op_reference/greater_equal.cpp [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+using namespace ngraph;
+using namespace InferenceEngine;
+using ComparisonTypes = ngraph::helpers::ComparisonTypes;
+
+namespace reference_tests {
+namespace ComparisonOpsRefTestDefinitions {
+namespace {
+TEST_P(ReferenceComparisonLayerTest, GreaterEqualCompareWithHardcodedRefs) {
+    Exec();
+}
```
[the remark this answers is not in the document]
Comment (pszmel) [reply docs/template_plugin/tests/functional/op_reference/greater_equal.cpp]: Removed.

**Review by dkozykowski — APPROVED** [review]

**Review by bszmelcz — APPROVED** [review]

File: docs/ops/comparison/GreaterEqual_1.md [hunk 2]
```diff
@@ -4,7 +4,16 @@
 
 **Category**: Comparison binary operation
 
-**Short description**: *GreaterEqual* performs element-wise comparison operation with two given tensors applying multi-directional broadcast rules.
+**Short description**: *GreaterEqual* performs element-wise comparison operation with two given tensors applying broadcast rules specified in the `auto_broadcast` attribute.
+
+**Detailed description**
+Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attributes is not `none`. Broadcasting is performed according to `auto_broadcast` value.
```
Comment (tsavina) [inline docs/ops/comparison/GreaterEqual_1.md:10]:
```suggestion
Before performing arithmetic operation, input tensors *a* and *b* are broadcasted if their shapes are different and `auto_broadcast` attribute is not `none`. Broadcasting is performed according to `auto_broadcast` value.
```

File: docs/ops/comparison/GreaterEqual_1.md [hunk 3]
```diff
@@ -4,7 +4,16 @@
...
+
+After broadcasting *GreaterEqual* does the following with the input tensors *a* and *b*:
```
Comment (tsavina) [inline docs/ops/comparison/GreaterEqual_1.md:12]:
```suggestion
After broadcasting, *GreaterEqual* does the following with the input tensors *a* and *b*:
```

File: docs/ops/comparison/GreaterEqual_1.md (hunk 2 above)
Comment (pszmel) [reply docs/ops/comparison/GreaterEqual_1.md]: Fixed.

File: docs/ops/comparison/GreaterEqual_1.md (hunk 3 above)
Comment (pszmel) [reply docs/ops/comparison/GreaterEqual_1.md]: Fixed.

**Review by ggalieroc — APPROVED** [review]

**Review by ilyachur — APPROVED** [review]

**Pull request merged**